rusk-recovery/config/mainnet.toml

```diff
@@ -0,0 +1,1928 @@
+amount = 3_000_000_000_000
+
+[[stake]]
+address = 'yWynRKzN1BdKJQDtv848LdSRJrFqdrudoDTnre2fiWAC8XvZuTb4UAjAkStu9idW4N9R8ncnuk4dsMFPE63AgF3Jbdemax5pH2YMkngxkjEBBaRpEV1wHHZh2ecS3vKeGiN'
+amount = 3_061_000_000_000
+
+[[stake]]
+address = 'yc8aiwLXGpXjHFgP6kgEHpTq6vewtqGHCNKdkvRndqeGsMrkxeEEX26FMY3sHFrMMxDyUSRFWbeWBaWkuMkErotcC4qhvDReKGddJ5g32G6parJsu43PBCnsvtumypnayqy'
+amount = 2_540_000_000_000
+
+[[stake]]
+address = 'qW2XxCLkoYZMHHonixqxLA5kitZNb7PhH2QsiVZfE1TPA6W2w7AMwgA6LPg4JE5k64j3xfgCX8WweGC5n8bjWbpbBEiiWdBEsATu1AyvRv2oVSy6qYtwYEkkNCC91qj8DW8'
+amount = 1_990_000_000_000
+
+[[stake]]
+address = '263iutoFfU2Hc3nRN3K3Jic5hBTL6dgPTzsavWfGRmgVzKp9g2uAmPmBnHPgej2i55Xrg5McFFW2Tp4EAJ51ukLTmUwRri4SE4ESKEkqPiEqV9xt1YUpKdFJ5TbU7a1rbroi'
+amount = 2_950_000_000_000
+
+[[moonlight_account]]
+address = '271zWxBm6QnM2qjfypWjE2h2RP7yYmc7nUqsozWCoZewDCUaBs86y7rmqCN6ejwoc6BPzrdu3erPBAev1yNRGdnVtemubxAQPPjzKK6bsHZrwKLPTHCJ8bZKXByaD4F9MFrk'
+balance = 1_000_000_000_000
+
+[[moonlight_account]]
+address = '262ZminCbPCFP6Cq5LfzWd8h58CJ1dRKKmV59RVYRhug4ymFFZ94sTzDqQhKKQj4Ef3a8SBAP1TcfM15HCGTEeaT3jHeFJwHQS3agTbcjVTcsPzfg5cgmsSLHkSBfJ3ret9Z'
+balance = 599_500_000_000_000
+
+[[moonlight_account]]
+address = 'zeSj4wpFJSXwysaXmaLQ4iL47dvCAEVdx1ricFWn96jDuUDbeMiGHrV9pQ8GNDvb4BbDVm3uN8tF7An9WfTvA98mSuVr2nKLr93dSkB3MPW6krCmy7yVr6fnjL3wbDgzWLR'
+balance = 50_000_000_000
+
+[[moonlight_account]]
+address = 'sZzMtHJFT9VjhoAkpj7sFrJc5T3rGX4oDBYgkYwJc2Xc2vTp45k1D8Lama6Ab5AYzyr3S9RBRtY2QsxaAjbw3dD4hq1JmFqVgpx45XqRGTYSonjM6tMb94fxoDB12DE42Le'
+balance = 37_000_000_000
+
+[[moonlight_account]]
+address = 'zzq8hzNY6Eonx8UYcZfBiQrrCPkn4CD6M9gYCdnMQhrK4KJvakS6PPwDuDBDcZ7enfqixt1UDxjxMGcNxo9FNdZMyJHBVYKiPBgrFK2pM9nC23cUQ9xnJTK4sNNWvfNkUtN'
+balance = 919_683_000_000_000
+
+[[moonlight_account]]
+address = '23ceBwdZrVeUvpV73bkMDscQYVBggWQDjgxi1jGfLrzQqPGPhcmx7Hw21Qq9woGimPaRezHMqo6i5ya2CN9yanizLMEj3Upu4AkAxLVUd4z5EMNeKsq8c2e6cScf4qJ93vxZ'
+balance = 461_000_000_000_000
+
+[[moonlight_account]]
+address = 'pcnLTUKwZmX8HdsVXmJnQvoTwieqvmvDnqre6iA9i8r78bEfKUs4D9GHXbJgayBR64r66mjQ8pCfVhY3BWk3X5jgE6KDjrWZdw7LawhiekMx9Ca1Edf9A9hQem8cB6Aqtf7'
+balance = 1_968_000_000_000
+
+[[moonlight_account]]
+address = 'zFVXTQZfY6gB2rWZFEm4ToWY2NKrpgeswTboWMAe7cjzaqHjsdGuADp8GsYzeruRkP3ZozA34mhWedEHkEiUr73iv8EcK3A192Y5b2uQYxLgprkZ2u8WKZyfDuZiyy4yEuL'
+balance = 500_000_000_000
+
+[[moonlight_account]]
+address = 'pqKjT4KYPL2c4Wdric9nZPL1WMefV5apFD8VMwXHAcZ6ZQbggoFXkumspWVoKJXzBgLWC2kbQk9Z6GUAdqTghetmTdfVGL7UmKn1gfWNh59AMuT5Q8dkBfxmmXbhknUFirF'
+balance = 108_976_765_380_000
+
+[[moonlight_account]]
+address = 'tAYWGiDnbB7MY7YckVXThPQ1SBLKjcggno3G6o9D5xgksQVHVtTgduZzi9hBj3FiruiAQtNoZyT699K4spAdGbHxBVYVCL814HxnoHrzzrUmRNC9HWya257EXrMLVaCZaA7'
+balance = 500_000_000_000
+
+[[moonlight_account]]
+address = 'od2bVPYstYud4N1GFQo9bAP2YmCz3Nxw1iE2hEdooVSRHyLnxufnXRdxZMNZnG2NH3YaCxDCVC46SuRn1ALA6fs8XvyGTxLoWPqhLBM3GyHusbDAVTprjtGdJS8FU8bEqvx'
```

Comment:
This user accidentally deposited instead of staked, and requested the following adjustment:
```toml
[[stake]]
address = 'od2bVPYstYud4N1GFQo9bAP2YmCz3Nxw1iE2hEdooVSRHyLnxufnXRdxZMNZnG2NH3YaCxDCVC46SuRn1ALA6fs8XvyGTxLoWPqhLBM3GyHusbDAVTprjtGdJS8FU8bEqvx'
amount = 5_600_000_000_000_000

[[moonlight_account]]
address = 'od2bVPYstYud4N1GFQo9bAP2YmCz3Nxw1iE2hEdooVSRHyLnxufnXRdxZMNZnG2NH3YaCxDCVC46SuRn1ALA6fs8XvyGTxLoWPqhLBM3GyHusbDAVTprjtGdJS8FU8bEqvx'
balance = 1_000_000_000_000
```